utils/NavigationItem.js

```diff
@@ -0,0 +1,27 @@
+const dfs = function ({ navigationItem, targetTitle }) {
```

Comment:
Could this be defined as a normal function i.e. `function dfs() {}`?
I'm not sure on the naming, I am guessing it stands for depth first search but I'm not sure that's obvious.

Reply:
Yep for sure, can rename and turn into a normal function!